src/common/metrics/src/monitor/in_mem.rs

```diff
@@ -24,7 +27,7 @@ pub type Count = Arc<AtomicU64>;
 pub struct CountMap(Arc<RwLock<HashMap<u64, Count>>>);
 
 impl CountMap {
-    pub(crate) fn new() -> Self {
+    pub fn new() -> Self {
         let inner = Arc::new(RwLock::new(HashMap::new()));
         #[cfg(all(not(test), not(madsim)))]
```

Comment:
Very weird... Buildkite is telling me that `tokio::spawn` below is stlll triggered https://buildkite.com/risingwavelabs/pull-request/builds/70906#0195b99e-5fbf-4d45-981b-4938ac94ff40?

But we have a conditional gate here which checks that..

Reply:
Any idea @xxchan? Wonder if we have some tricks in building the project, where we ignore some `cfg` flags.

Reply:
its compiled in unit tests. Seems expected  👀